report.py

```diff
@@ -0,0 +1,26 @@
+from telethon.tl.types import ChannelParticipantsAdmins
+
+
+async def init(bot):
+    # utils = modules['utils']
+
+    @bot.on(events.NewMessage(pattern='\/report'))
+    async def handler(event):
+        # mentions = '<a href="tg://user?id={}">{}</a> '.format(
+        #     event.sender_id,
+        #     utils.get_display(event.sender)
+        # )
+        if not event.reply_to_msg_id:
+            return
+        reply_message = await event.get_reply_message()
+        if not (
+            reply_message.media or 
+            reply_message.entities
```

Comment:
This allows someone to report messages with bold, italics, etc. text, which is not desirable.